src/bindings/python/src/openvino/runtime/properties/hint/overloads.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.utils import deprecatedclassproperty
+
+from openvino._pyopenvino.properties.hint import PerformanceMode as PerformanceModeBase
+
+
+class PerformanceMode(PerformanceModeBase):
+
+    @deprecatedclassproperty(
+        name="PerformanceMode.UNDEFINED",
+        version="2024.0",
```

Comment:
Shouldn't it be 2023.2?

Reply:
CPP counterpart:
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/properties.hpp#L302